test/helper.js

```diff
@@ -33,7 +33,7 @@ if (!cfg.get('azure:accountKey') ||
 var webServer = null;
 mocha.before(async () => {
   webServer = await server('test');
-  webServer.setTimeout(500);
+  webServer.setTimeout(5000);
```

Comment:
Explain this? This is webserver timeout of connections... killing them quickly means test process won't hang waiting for connections to close...